xc/device.ml

```diff
@@ -795,12 +792,14 @@ module NetSriovVf = struct
 
   let add  ~xs ~devid ~mac ?mtu ?(rate=None) ?(backend_domid=0) ?(other_config=[]) 
       ~pci ~vlan ~carrier ?(extra_xenserver_keys=[]) (task: Xenops_task.task_handle) domid =
-    let open Xenops_interface.Pci in
+    let dbg = Printf.sprintf "Task %s reference %s"
+      (Xenops_task.id_of_handle task) (Xenops_task.to_interface_task task).Task.dbg
+    in 
     let vlan_str = match vlan with None -> "none" | Some vlan -> sprintf "%Ld" vlan in
     let rate_str = match rate with None -> "none" | Some (a, b) -> sprintf "(%Ld,%Ld)" a b in
-    debug "Device.NetSriovVf.add domid=%d devid=%d pci=%s vlan=%s mac=%s carrier=%b \
+    debug "%s: Device.NetSriovVf.add domid=%d devid=%d pci=%s vlan=%s mac=%s carrier=%b \
```

Comment:
This isn't right: the `debug` command in this context already adds the `task.debug` string to all log lines, because it sits inside a `Debug.with_thread_associated` call.

Reply:
Fixed.